library/std/src/error.rs

```diff
@@ -470,6 +470,24 @@ impl Error for char::DecodeUtf16Error {
     }
 }
 
+#[unstable(feature = "map_try_insert", issue = "none")]
+impl<'a, K: Debug + Ord, V: Debug> Error
```

Comment:
I added a Error (and Display) implementations for OccupiedError for completeness, since it's an error. However, since it has non-static lifetimes, the Error implementation is not very useful in most cases.